plugins/obs-webrtc/whip-output.cpp

```diff
@@ -279,6 +284,57 @@ bool WHIPOutput::Setup()
 	return true;
 }
 
+void WHIPOutput::ParseLinkHeader(std::string val,
+				 std::vector<rtc::IceServer> &iceServers)
+{
+	size_t pos = 0;
+	std::string url, username, password;
+
+	auto extractUrl = [](std::string input) -> std::string {
+		auto head = input.find("<") + 1;
+		auto tail = input.find(">");
+
+		if (head == std::string::npos || tail == std::string::npos) {
+			return "";
+		}
+		return input.substr(head, tail - head);
+	};
+
+	auto extractValue = [](std::string input) -> std::string {
+		auto head = input.find("\"") + 1;
+		auto tail = input.find_last_of("\"");
+
+		if (head == std::string::npos || tail == std::string::npos) {
+			return "";
+		}
+		return input.substr(head, tail - head);
+	};
+
+	while ((pos = val.find(";")) != std::string::npos) {
```

Comment:
I think this won't process the last item if it doesn't have a semicolon which they don't in the linked example.

Reply:
Fixed. I now take rest of string on `npos` I tested with these values

```
< Link: <turn:192.168.1.93>; rel="ice-server"; username="username"; credential="password";
< Link: <turn:test>; rel="ice-server"; username="username"; credential="password"
```